app/views/admin/events/index.html.erb

```diff
@@ -33,31 +33,5 @@
 <%= paginate @events %>
 
 <p>
-  <%= link_to "Export Events to CSV", admin_export_events_url(format: "csv"), class: "usa-button export-btn", target: "_blank", rel: "noopener" %>
+  <%= link_to "Export Events to CSV", admin_export_events_url, class: "usa-button", target: "_blank", rel: "noopener" %>
```

Comment:
I know it's pre-existing behavior but it's kind of weird that this opens in a new tab when none of the other exports do.